src/backend/routeHandler.ts

```diff
@@ -0,0 +1,81 @@
+import bodyParser from 'body-parser';
+
+import {PluginConfig, PluginRouteOptions} from '../@types/plugin';
+
+export = function configureRoutes(options: PluginRouteOptions<PluginConfig>): void {
+  options.router.use(bodyParser.json({limit: '100mb', extended: true} as any));
+
+  options.router.get('/getSchema', async (req, res) => {
+      try {
+          const schemaResult = await options.getRestClient(req).graphSchema.getTypesWithAccess({
+              entityType: req.query.entityType || 'node' as any,
+              sourceKey: req.query.sourceKey as string
+          });
+          res.status(200);
+          res.contentType('application/json');
+          res.send(JSON.stringify(schemaResult));
+      } catch (e) {
+          res.status(412);
+          res.send(JSON.stringify({status: 412, body: e}));
+      }
+  });
+
+  options.router.post('/addNodes', async (req, res) => {
+      try {
+          if(!req.body || !req.body.nodes || !Array.isArray(req.body.nodes)) {
+              throw new Error('Payload is empty.');
+          }
+          let nodesPromise: Promise<any>[] = [];
+          req.body.nodes.forEach((node: any) => {
+              if(!node || !node.categories || !node.properties) {
+                  throw new Error('Node parameters are invalid or missing.');
+              }
+              nodesPromise.push(options.getRestClient(req).graphNode.createNode({
+                  categories: node.categories,
+                  properties: node.properties,
+                  sourceKey: req.query.sourceKey as string
+              }));
+          });
+          let totalSuccessful = 0, totalFailed = 0;
+          const results = await Promise.all(nodesPromise);
```

Comment:
we tried but node version is too low if I remember correctly. We added a npm package but it was creating errors with TS, so for now I did it like this. Knowing that the backend part is secondary for now, since it will be re-written for the second iteration (on feedbacks)